src/test/user.spec.js

```diff
@@ -674,3 +675,111 @@ describe('PATCH /api/v1/auth/user/:id/role', () => {
     expect(userRoleStatus.length).to.equal(1);
   });
 });
+
+describe(' POST /api/v1/auth/reset-password', async () => {
+  const newUser = await createUser();
```

Comment:
I think this test for `reset password` should not come along with the task `(report an article)`

Reply:
i dont know it get in here